rules/integrations/o365/initial_access_microsoft_365_impossible_travel_activity.toml

```diff
@@ -1,51 +0,0 @@
-[metadata]
-creation_date = "2021/07/15"
-integration = ["o365"]
-maturity = "development"
-updated_date = "2023/06/22"
-
-[rule]
-author = ["Austin Songer"]
-description = """
-Identifies when a Microsoft Cloud App Security reported a risky sign-in attempt due to a login associated with an
-impossible travel.
-"""
-false_positives = ["User using a VPN may lead to false positives."]
-from = "now-30m"
-index = ["filebeat-*", "logs-o365*"]
-language = "kuery"
-license = "Elastic License v2"
-name = "Microsoft 365 Impossible travel activity"
```

Comment:
maybe instead of deleting the rule just add a note with the link to Microsoft retiring the anomaly event https://learn.microsoft.com/en-us/defender-cloud-apps/cloud-discovery-anomaly-detection-policy and referring to the new rule.ids

Reply:
Sure thing, Ill add that.